### DISCUSSION TITLE
[CLOSED] Is the json latest abandonned

### DISCUSSION THREAD

**Discussion by rockline69** [owid/covid-19-data#2189] [Q&A] [answered]
Hello all,
I tried to check the data from the link below but it seems that the data has not been updated since January 2021. Is this json being abandonned?
https://covid.ourworldindata.org/data/latest/owid-covid-latest.json
Thank you.

**Comment by lucasrodes** [comment, marked as answer]
Hi @rockline69,
Thanks for your message.

I just checked and the link appears to work as usual, with data from yesterday (i.e. 2021-12-19).

**Reply by rockline69** [reply]
hi Lucas, here is what I've got:
![image](https://user-images.githubusercontent.com/73796998/146758221-190b7623-787c-4231-a6e1-2c6ce8e418fe.png)
![image](https://user-images.githubusercontent.com/73796998/146758356-bf32af16-c5ee-491d-9d70-8218e06767b1.png)

**Reply by lucasrodes** [reply]
What process are you following to download this data?

It shows data for 2021-12-19 for me at https://covid.ourworldindata.org/data/latest/owid-covid-latest.json

![image](https://user-images.githubusercontent.com/18101289/146766779-68e3ffd0-37f6-46d6-ae22-b35cf64eb733.png)
![image](https://user-images.githubusercontent.com/18101289/146766847-e3ba0e89-0ddd-4b83-bacd-f107ba34dd38.png)

**Reply by rockline69** [reply]
This is very strange indeed. I'm using excel (Data-->NewQuery-->FromWeb) to visualize it on my PC as well as my own Android app on my phone via the JSONObject class. I wonder if there is a caching issue somehow in both cases.

**Reply by lucasrodes** [reply]
That might be the case, I believe. Is the data showing up to date if you use the web browser?

**Reply by rockline69** [reply]
I've just tried with a private window in Firefox and it seems to display what you're showing so clearly I've got an issue with the caching. Thanks again for your help. Now the challenge will be to figure out how caching works in Android.
